crates/ide_assists/src/utils/gen_trait_fn_body.rs

```diff
@@ -225,10 +221,8 @@ fn gen_debug_impl(adt: &ast::Adt, func: &ast::Fn) -> Option<()> {
 /// Generate a `Debug` impl based on the fields and members of the target type.
 fn gen_default_impl(adt: &ast::Adt, func: &ast::Fn) -> Option<()> {
     fn gen_default_call() -> ast::Expr {
-        let trait_name = make::ext::ident_path("Default");
-        let method_name = make::ext::ident_path("default");
-        let fn_name = make::expr_path(make::path_concat(trait_name, method_name));
-        make::expr_call(fn_name, make::arg_list(None))
+        let fn_name = make::path_from_text(&"Default::default");
```

Comment:
`path_from_text` shouldn't be used, it being pub is actually wrong(Note that FIXME on it, I'll make it priv later), let's do something else in the `make::ext` module that takes an `impl IntoIterator<Item = &str>` and constructs the path that way.

Otherwise lgtm
bors d+

Reply:
ahaha, dang -- that's what I had right before realizing we could use `path_from_text`. time to press "undo" a bunch on my keyboard I guess :P